pages/app/index.ftl

```diff
@@ -2,8 +2,12 @@
 
 <@tmpl.page current="test-app" title="Test application" noindex=true nocsp=true>
 
-<script src="keycloak.js" type="text/javascript"></script>
-<script src="app.js" type="text/javascript"></script>
+<#if useLegacyKeycloakJs>
```

Comment:
I would have just done this, so there is no changes needed in the global attributes:

```suggestion
<#if version.majorVersion < 26>
```

Doesn't really matter as we will remove this stuff after the 26 release.